client/src/components/Game/Board.js

```diff
@@ -77,31 +122,30 @@ const Card = ({status, color, word, onClick}) => {
   );
 };
 
-const Board = ({ state, setState, gameID, onNextMove }) => {
+const Board = ({ state, setState, timer, gameID, onNextMove, getRole }) => {
```

Comment:
We're starting to pass in a lot of props, could we use `propTypes` here to catch any propagation errors?